crates/bevy_asset_store/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "bevy_asset_store"
+version = "0.1.0"
+edition = "2021"
+
+[dependencies]
+async-stripe = "0.39.1"
```

Comment:
I feel like this might be a controversial inclusion. Artists should definitely be paid for their work, but I don't know if the Bevy Foundation wants to be involved in developer-artist payment processing. I think it would be a better focus for the editor to work with user configurable asset stores (e.g., a local folder, a remote server using some REST API, etc.). If the Bevy Foundation wants to setup a paid store at some point in the future then the editor wont need any additional code changes to support it.

Reply:
Yep, as discussed this should be cut: we'll let the webstore handle payments.